konig-schemagen-maven-plugin/src/main/java/io/konig/maven/KonigSchemagenMojo.java

```diff
@@ -520,7 +526,7 @@ private void loadResources() throws MojoExecutionException, RDFParseException, R
     		defaults = new RdfConfig();
     	}
     	GcpShapeConfig.init();
-    	
+    	OracleShapeConfig.init();

```

Comment:
We should create a static method somewhere that invokes GcpShapeConfig.init() and OracleShapeConfig.init().  Otherwise, it might get tricky trying to track down all of the configurations that must be initialized.